server/app/views/_partials/tracking-lateral.hbs

```
@@ -0,0 +1,17 @@
+<script>
+(function() {
+	window.trackLateralPageView = function() {
+		if (!M.prop('queryParams.noexternals')) {
+			(function(i) {
+				var a = document.location.pathname.replace(/\/+$/,"").split("/").pop(),
+					b = RegExp("wikia_beacon_id=([A-Za-z0-9_-]{10})").exec(document.cookie);
```

If failing silently is fine then :+1: 
We had some errors thrown but it's most likely a matter of the code around.